tox.ini

```diff
@@ -7,15 +7,16 @@ envlist =
     # the version matrix according to the Django docs:
     # https://docs.djangoproject.com/en/2.0/faq/install/#what-python-version-can-i-use-with-django
     py{27,36}-dj111-tests,
-    py{36,37}-dj{21,22}-tests,
+    py{36,37,38}-dj{21,22}-tests,
     py{27,36}-fl{011,012,10}-tests,
-    py{36,37}-s19-tests,
+    py{36,37,38}-s19-tests,
 
 [testenv]
 basepython =
     py27: python2.7
     py36: python3.6
     py37: python3.7
+    py38: python3.8
```

Comment:
Note: maybe we should also consider changing `[testenv:py36-lint]` below.

Reply:
As well as `tests-py{27,34}: PYTEST_ADDOPTS = --ignore=tests/test_sanic.py`.

Reply:
Ah yeah sorry read that too quickly

Reply:
Oh, woops I've been doing those changes locally as well but ran against some issues with the test setup. Will incorporate yours changes